kaldb/src/main/proto/trace.proto

```diff
@@ -67,6 +71,8 @@ message Span {
 
   // A list of key value pairs.
   repeated KeyValue tags = 7;
+
+  string service_name = 8;
```

Comment:
Is there a reason to make `service_name` a first class citizen and not a tag?

Reply:
fixed!